lib/interface.ts

```diff
@@ -52,7 +51,8 @@ export interface IOurRequest extends restify.Request {
     clientCert?: any;
     blpSession: BAPI.Session;
     apiSession?: IAPISession;
-    identity?: blpapi.Identity;
+    identity?: BAPI.IIdentity;
+    label?: string;
```

Comment:
I don't think we are exposing the `label` in the http project yet. Let's leave it out until we have a real usage of it.